content/_code-samples/quickstart/js/8.escrow.html

```diff
@@ -0,0 +1,305 @@
+                          </tr>
+                          <tr>
+                          <tr>
+                            <td align="right">
+                              Escrow Sequence Number
+                            </td>
+                            <td>
+                              <input type="text" id="operationalEscrowSequenceField" size="40"></input>
+                              <br>
+                            </td>
+                          </tr>
+                          <tr>                            <td align="right">
+                              Transaction to Look Up
+                            </td>
+                            <td>
+                              <input type="text" id="operationalTransactionField" size="40"></input>
+                              <br>
+                            </td>
+                          </tr>
+                          <tr>
+                          <td>
+                            </td>
+                            <td align="right">
+                            <input type="checkbox" id="operationalDefault" checked="true"/>
+                              <label for="operationalDefault">Allow Rippling</label>
+                              <button type="button" onClick="configureAccount('operational',document.querySelector('#operationalDefault').checked)">Configure Account</button>
+                            </td>
+                          </tr>
+                          <tr>
+                            <td align="right">
+                              Public Key
+                            </td>
+                            <td>
+                              <input type="text" id="operationalPubKeyField" size="40"></input>
+                              <br>
+                            </td>
+                          </tr>
+                          <tr>
+                            <td align="right">
+                              Private Key
+                            </td>
+                            <td>
+                              <input type="text" id="operationalPrivKeyField" size="40"></input>
+                              <br>
+                            </td>
+                          </tr>
+                          <tr>
+                            <td align="right">
+                              Seed
+                            </td>
+                            <td>
+                              <input type="text" id="operationalSeedField" size="40"></input>
+                              <br>
+                            </td>
+                          </tr>
+                        </table>
+                        <p align="right">
+                          <textarea id="operationalResultField" cols="80" rows="20" ></textarea>
+                        </p>
+                      </td>
+                      </td>
+                    </tr>
+                    </td>
+                    </tr>
+                  </table>
```

Comment:
There are more duplicate/extra close tags here. I believe the following fixes it:

```suggestion
                      </td>
                    </tr>
                  </table>
```